develop/__utils/enum.util.ts

```diff
@@ -0,0 +1,4 @@
+export const enum sortDirectionTypes {
```

Comment:
why is this a separate file?

Reply:
Because we don't have an enum util file of course . It is an an enum which supposedly belong in an enum file hence the reason why it's separate.